concent_api/core/tests/test_integration_report_computed_task.py

```diff
@@ -1835,54 +1834,6 @@ def test_provider_forces_computed_task_report_empty_key_returns_400_error(self):
 
         self._assert_client_count_is_equal(0)
 
-    def test_requestor_sends_ack_report_computed_task_with_message_cannot_compute_task(self):
```

Comment:
and this? another validation in GM?

Reply:
Yup, GM now contains validation for wrong nested message.